src/components/Alert.tsx

```diff
@@ -0,0 +1,64 @@
+import { IonAlert } from '@ionic/react';
+import { createContext, useContext, useState } from 'react';
+
+interface IAlertData {
```

Comment:
이 부분도 export 해주시고, 
```suggestion
export default interface IAlertData {
```

`src/types/IAlertData.d.ts`로 분리시켜주세용
`.ts`가 아니라  `.d.ts`입니다~